check-code-style/action.yml

```diff
@@ -11,102 +11,60 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Install clang-format-13 if Ubuntu
+    - name: Install code checking tools if on Ubuntu
       if: inputs.os == 'ubuntu-latest'
       run: |
-        # Install Clang 13 (including clang-format-13) through LLVM's preferred mechanism:
+        # Install Clang (including clang-format) through LLVM's preferred mechanism:
         #   https://apt.llvm.org/
+        CLANG_VERSION=14
         wget https://apt.llvm.org/llvm.sh
         chmod +x llvm.sh
-        sudo ./llvm.sh 13
+        sudo ./llvm.sh ${CLANG_VERSION}
         rm ./llvm.sh
-        sudo apt-get install clang-format-13
+        sudo apt-get install -y clang-format-${CLANG_VERSION}
         sudo rm -rf /var/lib/apt/lists/*
         # Remove existing symlink to clang-format and replace it.
         sudo rm /etc/alternatives/clang-format
-        sudo ln -s /usr/bin/clang-format-13 /etc/alternatives/clang-format
-      shell: bash
+        sudo ln -s /usr/bin/clang-format-${CLANG_VERSION} /etc/alternatives/clang-format
 
-    # On macOS we should install clang-format.
-    - name: Install clang-format-13 if macOS
-      if: inputs.os == 'macos-latest'
-      run: |
-        # TODO(benh): is 'brew' not in PATH on macOS like it is not in
-        # the PATH for Linux? If so, where does GitHub install 'brew'
-        # on a macOS runner?
-        brew install clang-format@13
-      shell: bash
+        # Install buildifier.
+        BUILDIFIER_VERSION=5.1.0
```

Comment:
I think we want to take a different approach in our `respect` repo: Instead of using this action, running on a `ubuntu:latest` we'll use our codespace runner (built with our `Dockerfile`) to just run `check-style.sh --full`.

This will save the built time (1m30s of 2m on every go) and will also ensure that the version always stays the same.